posts/navarroaxel/06.how-to-use-databases-in-the-Docker-world.md

```diff
@@ -0,0 +1,182 @@
+---
+title: How to use databases in the Docker world
+published: true
+description: Run DB engines locally on our machine without installing them by using Docker containers.
+tags: 'docker, database, mongodb, postgres'
+cover_image: ./assets/sign-your-code-cover.png
+id: 1059901
+---
+
```

Comment:
I'm not sure if it's worth but could be interesting to mention that this fits great for local development but is not a desirable solution for production environments